src/CSET/operators/ageofair.py

```diff
@@ -0,0 +1,378 @@
+# Copyright 2024 Met Office and contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Age of air operator.
+
+This determines how old air is since it entered through the lateral boundary at some given leadtime.
+"""
+
+import datetime
+import logging
+import multiprocessing
+import os
+import tempfile
+from functools import partial
+from math import atan2, cos, radians, sin, sqrt
+
+import iris
+import iris.cube
+import numpy as np
+from scipy.ndimage import gaussian_filter
+
+
+def calc_dist(coord_1, coord_2):
+    """Haversine distance in meters."""
+    # Approximate radius of earth in km
```

Comment:
I presume the difference between the sphere assumed here and the actual shape of the earth is minimal relative to the uncertainty in the wind?

Reply:
Google is mixed, but I see "A globally-average value is usually believed to be 6,371 kilometres (3,959 mi) with a 0.3% variability (±10 km) for the following reasons." I am using the value from NASA website of 6378. While error is small, important thing is that is applied consistently and diagnostic is currently used as a qualitative tool